docs/src/consensus/block.md

```diff
@@ -37,24 +37,40 @@ The genesis block is hardcoded in the source code, see `genesis_block` in `block
 
 ### B: Incremental Validity
 
-A block is incrementally valid if (all of):
- - ***a)*** the transaction is valid
- - ***b)*** the transaction's coinbase conforms with the block subsidy schedule
- - ***c)*** all the inputs in the transaction either live in the lock-free UTXO MMR or have at least one index that is absent from the mutator set SWBF
- - ***d)*** the `mutator_set_accumulator` results from applying all removal records and then all addition records to the previous block's `mutator_set_accumulator`
- - ***e)*** the `block_mmr_accumulator` results from appending the previous block's hash to the previous block's `block_mmr_accumulator`
- - ***f)*** there is an ancestor block `luca` of the current block such that for each uncle block `uncle`
-   - `uncle` is valid
-   - `luca` is an ancestor of `uncle`
-   - neither `luca` nor any of the blocks between `luca` and the current block list `uncle` as an uncle block
- - ***g)*** the `version` matches that of its predecessor or is member of a predefined list of exceptions
- - ***h)*** the `height` is one greater than that of its predecessor
- - ***i)*** the `timestamp` is greater than that of its predecssor
- - ***j)*** the network statistics trackers are updated correctly
- - ***k)*** the variable network parameters are updated correctly.
+A block is incrementally valid, relative to a predecessor block, iff (all of):
```

Comment:
What does "incrementally valid" mean? Does it mean that its validity depends on its parent's validity?